frontend/src/index.jade

```diff
@@ -126,11 +126,12 @@ html
           .author {{ info.name }} ({{ info.author }}) contribution from {{ info.minDate }} to {{ info.maxDate }}
           .contribution(v-if="isLoaded")
             span Total LoC contributed:&nbsp;
-            span.loc {{ totalLineCount }}
+            span.loc {{ totalLineCount }}&nbsp;
+            span.bloc (Blank: {{ totalBlankLineCount }}, Non-Blank: {{ totalLineCount - totalBlankLineCount }})
             br
             span.fileType(v-if="totalLineCount > 0") &rarr; &nbsp;
                 template(v-for="type in Object.keys(filesLinesObj)")
-                    span {{ type }}: {{ filesLinesObj[type] }}
+                    span(v-bind:title="getBlankLineCount(type)") {{ type }}: {{ filesLinesObj[type] }}
```

Comment:
Cant seem to see this portion on the netlify deploy?

Reply:
it appears when you hover over the filetype in the code view in netlify